### ISSUE TITLE
How did you learn of zVirtualDesktop?

### ISSUE THREAD

**Issue by mzomparelli** [mzomparelli/zVirtualDesktop#128]
I'm trying to get an idea of how you guys found this program so that I can work on making it more accessible. Please, in as few words as you can, tell me how you discovered zVirtualDesktop.

**Comment by nicolasdejong** [comment]
I got here through a comment on StackOverflow. Software grows best in the shadow, so I don't mind that it is not yet so well known. Eventually it'll happen, if there are enough features to trigger folks.

**Comment by fluffynuts** [comment]
TL;DR
A friend of mine pointed me here. 

More info:
We have both used VirtuaWin for literally years, and both of us have tried out alternatives, always coming back to VirtuaWin (which doesn't seem to be very active any more) simply because VirtuaWin always _simply worked_ where others might have more bells and whistles but would either have issues or cost too much.

He brought it to my attention because we both knew about the native virtual desktops in win10 and we'd already discussed me writing software like zVirtualDesktop. At the time that we had discussed it, the api for win10 virtual desktops was very uninspiring and there were several technical hurdles that I didn't feel I had the time or energy to circumvent -- and VirtuaWin was still working well enough. In particular, pinning apps to desktops and pre-defining a set number of desktops were blockers for me (and I still really want a way for zVirtualDesktop to remember this between app sessions)

**Comment by jjhidalgar** [comment]
I simply looked for "virtual desktop" windows 10 on github and tested every single proyect to find something interesting. I had in mind something like a linux like taskbar widget to switch clicking if I remember but I found this one.

**Comment by dsp0105** [comment]
"virtual desktop" windows 10 on github

-- learnt a good technique to search - thank you :)
